core-web/libs/sdk/client/src/lib/client/content/methods/get-collection/index.ts

```diff
@@ -0,0 +1,146 @@
+import { QueryBuilder } from '../../../../query-builder/sdk-query-builder';
+import { Equals } from '../../../../query-builder/utils/lucene-syntax/Equals';
```

Comment:
Not a fan of this folder structure, seems very complex to me

Reply:
I changed a bit